### PR TITLE
Add ability to configure `SwaggerJson` via params

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️/❌
| Fixed issues  | -

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Review by vjik — COMMENTED** [review]
Need changelog and info about new params in readme.

**Comment by samdark** [comment]
@rustamwin need a line for CHANGELOG since it's a released package.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Yii Swagger Change Log
 
-## 1.0.1 under development
+## 1.1.0 under development
 
-- no changes in this release.
+- Enh #48: Add ability to configure `Yii\Swagger\Middleware\SwaggerJson` via config params (@rustamwin)
```
Comment (vjik) [inline CHANGELOG.md:5]:
```suggestion
- Enh #48: Add ability to configure `Yii\Swagger\Middleware\SwaggerJson` via config params (rustamwin)
```

File: README.md [hunk 2]
```diff
@@ -90,7 +83,23 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
 See [Swagger-PHP documentation](https://zircote.github.io/swagger-php/Getting-started.html#write-annotations) for details
 on how to annotate your code.
 
-### 3. (Optional) Add config for aliases and asset manager
+### 3. Configure SwaggerJson
```
Comment (vjik) [inline README.md:86]:
```suggestion
### 3. Configure `SwaggerJson` middleware
```

File: README.md [hunk 3]
```diff
@@ -90,7 +83,23 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
...
+
+For annotations to be registered you need to configure `SwaggerJson`.
+
+You can use the parameters in `config/params.php` to configure SwaggerJson:
```
Comment (vjik) [inline README.md:90]:
```suggestion
You can use the parameters in `config/params.php` to configure `SwaggerJson`:
```

File: README.md [hunk 4]
```diff
@@ -90,7 +83,23 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
...
+
+```php
+//...
+'yiisoft/yii-swagger' => [
+    'annotation-paths' => [
+        '@src/Controller' // Directory where annotations are used
+    ]
```
Comment (vjik) [inline README.md:97]: Maybe also add `cacheTTL`  here?

File: README.md [hunk 5]
```diff
@@ -115,7 +124,7 @@ return [
     //...
 ```
 
-### 4. (Optional) Configure SwaggerUI
+### 5. (Optional) Configure SwaggerUI
```
Comment (vjik) [inline README.md:127]:
```suggestion
### 5. (Optional) Configure `SwaggerUi` middleware
```

File: README.md [hunk 6]
```diff
@@ -115,7 +124,7 @@ return [
...
 
 You can use the parameters in `config/params.php` to configure SwaggerUI.
```
Comment (vjik) [inline README.md:129]:
```suggestion
You can use the parameters in `config/params.php` to configure `SwaggerUi`.
```

File: config/web.php [hunk 7]
```diff
@@ -14,4 +16,12 @@
             'params' => $params['yiisoft/yii-swagger']['ui-params'],
         ],
     ],
+    SwaggerJson::class => static function (Injector $injector) use ($params) {
```
Comment (vjik) [inline config/web.php:19]: `Injector` is transitive dependency. Better use dependencies of `SwaggetJson` instead `Injector`.

For this case, solution of issue https://github.com/yiisoft/config/issues/113 can help in future.

Comment (rustamwin) [reply config/web.php]: Done.

**Review by vjik — APPROVED** [review]

**Pull request merged**